### PR TITLE
fix outdated validatingwebhook documentation

### PR DESCRIPTION
**Pull request by kfaseela**
Signed-off-by: Faseela K <faseela.k@est.tech>

The validatingwebhookconfig name is wrong in the document. Also the document talks about a `istio-validation` ConfigMap, but cannot see such a ConfigMap when deploying istio.

Fixes: #12220

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by zirain — APPROVED** [review]
LGTM

File: content/en/docs/ops/common-problems/validation/index.md [hunk 1]
```diff
@@ -23,60 +23,60 @@ necessary.
 
 ## Invalid configuration is accepted
 
-Verify the `istiod-istio-system` `validationwebhookconfiguration` exists and
-is correct. The `apiVersion`, `apiGroup`, and `resource` of the
-invalid configuration should be listed in one of the two `webhooks`
-entries.
+Verify the `istio-validator` `validationwebhookconfiguration` exists and
```
Comment (frankbu) [inline content/en/docs/ops/common-problems/validation/index.md:26]:
```suggestion
Verify the `validatingwebhookconfiguration` exists and
```

File: content/en/docs/ops/common-problems/validation/index.md [hunk 2]
```diff
@@ -23,60 +23,60 @@ necessary.
...
+is correct. Please note that the name of `validationwebhookconfiguration` will be
+`istio-validator-` suffixed by your revision and Istio namespace.
```
Comment (frankbu) [inline content/en/docs/ops/common-problems/validation/index.md:28]:
```suggestion
Verify that a `validatingwebhookconfiguration` named `istio-validator-` followed by
`<revision>-`, if not the default revision, followed by the istio namespace
(e.g., `istio-validator-myrev-istio-system`) exists and is correct.
```

File: content/en/docs/ops/common-problems/validation/index.md [hunk 3]
```diff
@@ -86,51 +86,7 @@ webhooks:
...
-        - "*"
-        resources:
-        - "*"
-    failurePolicy: Fail
-    sideEffects: None]
-      (... snip ...)
+  timeoutSeconds: 10
 {{< /text >}}
 
 If the webhook array in `istio-validation` is empty, verify
```
Comment (frankbu) [inline content/en/docs/ops/common-problems/validation/index.md:92]:
```suggestion
If the webhook array in `istio-validator-` is empty, verify
```

File: content/en/docs/ops/common-problems/validation/index.md [hunk 4]
```diff
@@ -86,54 +86,10 @@ webhooks:
...
-    failurePolicy: Fail
-    sideEffects: None]
-      (... snip ...)
+  timeoutSeconds: 10
 {{< /text >}}
 
-If the webhook array in `istio-validation` is empty, verify
+If the webhook array in `istio-validator-` is empty, verify
 the `global.configValidation` installation options are
 set.
```
Comment (frankbu) [inline content/en/docs/ops/common-problems/validation/index.md:94]: Actually, it looks to me like the webhook isn't applied at all if global.configValidation is not set.

```suggestion
If the `istio-validator-` webhook does not exist, verify
the `global.configValidation` installation option is
set to `true`.
```

File: content/en/docs/ops/common-problems/validation/index.md [hunk 5]
```diff
@@ -23,60 +23,60 @@ necessary.
 
 ## Invalid configuration is accepted
 
-Verify the `istiod-istio-system` `validationwebhookconfiguration` exists and
-is correct. The `apiVersion`, `apiGroup`, and `resource` of the
-invalid configuration should be listed in one of the two `webhooks`
-entries.
+Verify that a `validatingwebhookconfiguration` named `istio-validator-` followed by
+`<revision>-`, if not the default revision, followed by the istio namespace
```
Comment (frankbu) [inline content/en/docs/ops/common-problems/validation/index.md:27]:
```suggestion
`<revision>-`, if not the default revision, followed by the Istio system namespace
```

**Review by frankbu — APPROVED** [review]

**Comment by zirain** [comment]
@frankbu seems still a flaky in `doc.test.profile_default`

**Pull request merged**